docs/content/docs/connectors/table/kafka.md

```diff
@@ -217,7 +217,7 @@ Connector Options
       <td>required by source</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>The id of the consumer group for Kafka source, optional for Kafka sink.</td>
+      <td>The id of the consumer group for Kafka source, optional for Kafka sink. If group ID is not specified, an automatically generated id "KafkaSource-{tableIdentifier}" will be used.</td>
```

Comment:
There's seems to be some inconsistency in the doc. Let's be clear about whether it's required by source.